app/routes/wallet/credentials.py

```diff
@@ -1,34 +1,59 @@
-from typing import Optional
+from typing import List, Optional
 
 from aries_cloudcontroller import (
     AttributeMimeTypesResult,
-    CredInfoList,
     CredRevokedResult,
-    IndyCredInfo,
-    VCRecord,
-    VCRecordList,
     W3CCredentialsListRequest,
 )
-from fastapi import APIRouter, Depends
+from fastapi import APIRouter, Depends, Query
 
 from app.dependencies.acapy_clients import client_from_auth
 from app.dependencies.auth import AcaPyAuth, acapy_auth_from_header
 from app.exceptions import handle_acapy_call
+from app.models.wallet import CredInfoList, IndyCredInfo, VCRecord, VCRecordList
 from shared.log_config import get_logger
 
 logger = get_logger(__name__)
 
 router = APIRouter(prefix="/v1/wallet/credentials", tags=["wallet"])
 
 
-@router.get("", response_model=CredInfoList)
+@router.get(
+    "",
+    response_model=CredInfoList,
+    summary="Fetch a list of credentials from the wallet",
+)
 async def list_credentials(
     count: Optional[str] = None,
     start: Optional[str] = None,
     wql: Optional[str] = None,
     auth: AcaPyAuth = Depends(acapy_auth_from_header),
 ) -> CredInfoList:
-    """Fetch a list of credentials from the wallet."""
+    """
+    Fetch a list of credentials from the wallet
+    ---
+
+    The `wql` (Wallet Query Language) parameter can be used to filter credentials returned from the wallet.
+
+    The following string will look for the credential with the attribute `age` with value `21`:
+
+        {"attr::age::value": "21"}
+
+    Optional Parameters:
+    ---
+        count: str
+            The number of records to return.
+        start: str
+            The number of records to skip before starting to return records.
+        wql: str
+            A WQL query to filter records.
+
+    Returns:
+    ---
+        CredInfoList
+            A list of credential records.
+
+    """
     logger.info("GET request received: List credentials")
```

Comment:
ah, just check that we rebase this to be .debug() log levels - I think all of the received/success logs in this module should just be debug level